panel/src/helpers/field.js

```diff
@@ -9,7 +9,7 @@ export function form(fields) {
 	const form = {};
 
 	for (const fieldName in fields) {
-		form[fieldName] = clone(fields[fieldName].default);
+		form[fieldName] = clone(fields[fieldName].default ?? null);
```

Comment:
⚠️ This is a not solution idea!

I changed it `null` fallback (instead `undefined`) to give you an idea about the issue. Because `{name: undefined}` object stored as `{}`. I couldn't understand it.